gsa/src/web/components/dashboard/dashboard.js

```diff
@@ -96,7 +96,7 @@ const RowPlaceHolder = styled.div`
 
 export const Dashboard = props => {
   const getSetting = useGetSetting();
-  const {data} = getSetting({
+  const {data, loading} = getSetting({
```

Comment:
If this is a boolean, it should be named ´´´isLoading´´´.

Reply:
Dashboard gets an isLoading prop already, passed from a lot of other dashboards that do not have the new backend. I don't know what will happen if I touched that right now. loading is an Apollo exported variable.